flow/designs/bigger_is_better_metrics.json

```diff
@@ -0,0 +1,178 @@
+    "cts__design__instance__utilization__stdcell__post_repair",
+    "cts__design__instance__utilization__stdcell__pre_repair",
+    "cts__design__io",
+    "cts__design__io__post_repair",
+    "cts__design__io__pre_repair",
+    "cts__design__violations",
+    "cts__power__internal__total",
+    "cts__power__internal__total__post_repair",
+    "cts__power__internal__total__pre_repair",
+    "cts__power__leakage__total",
+    "cts__power__leakage__total__post_repair",
+    "cts__power__leakage__total__pre_repair",
+    "cts__power__switching__total",
+    "cts__power__switching__total__post_repair",
+    "cts__power__switching__total__pre_repair",
+    "cts__power__total",
+    "cts__power__total__post_repair",
+    "cts__power__total__pre_repair",
+    "cts__timing__drv__max_cap",
+    "cts__timing__drv__max_cap__post_repair",
+    "cts__timing__drv__max_cap__pre_repair",
+    "cts__timing__drv__max_cap_limit",
+    "cts__timing__drv__max_cap_limit__post_repair",
+    "cts__timing__drv__max_cap_limit__pre_repair",
+    "cts__timing__drv__max_fanout",
+    "cts__timing__drv__max_fanout__post_repair",
+    "cts__timing__drv__max_fanout__pre_repair",
+    "cts__timing__drv__max_fanout_limit",
+    "cts__timing__drv__max_fanout_limit__post_repair",
+    "cts__timing__drv__max_fanout_limit__pre_repair",
+    "cts__timing__drv__max_slew",
+    "cts__timing__drv__max_slew__post_repair",
+    "cts__timing__drv__max_slew__pre_repair",
+    "cts__timing__drv__max_slew_limit",
+    "cts__timing__drv__max_slew_limit__post_repair",
+    "cts__timing__drv__max_slew_limit__pre_repair",
+    "cts__timing__setup__ws",
+    "cts__timing__setup__ws__post_repair",
+    "cts__timing__setup__ws__pre_repair",
+    "detailedplace__design__instance__count",
+    "detailedplace__design__instance__count__macros",
+    "detailedplace__design__instance__count__stdcell",
+    "detailedplace__design__instance__displacement__max",
+    "detailedplace__design__instance__displacement__mean",
+    "detailedplace__design__instance__displacement__total",
+    "detailedplace__design__instance__utilization",
+    "detailedplace__design__instance__utilization__stdcell",
+    "detailedplace__design__io",
+    "detailedplace__design__violations",
+    "detailedplace__power__internal__total",
+    "detailedplace__power__leakage__total",
+    "detailedplace__power__switching__total",
+    "detailedplace__power__total",
+    "detailedplace__timing__drv__max_cap",
+    "detailedplace__timing__drv__max_cap_limit",
+    "detailedplace__timing__drv__max_fanout",
+    "detailedplace__timing__drv__max_fanout_limit",
+    "detailedplace__timing__drv__max_slew",
+    "detailedplace__timing__drv__max_slew_limit",
+    "detailedplace__timing__setup__ws",
+    "detailedroute__route__drc_errors",
+    "detailedroute__route__drc_errors__iter__1",
+    "detailedroute__route__drc_errors__iter__2",
+    "detailedroute__route__drc_errors__iter__3",
+    "detailedroute__route__drc_errors__iter__4",
```

Comment:
@habibayassin 
These we want to be smaller. BTW, the wild card did not work? This would be a great place to use it.

Reply:
I must have not added the errors to wild cards in the generating script, I will push a new file with the edits now. Do you prefer the file in that format or
`["*displacement*",
"*utilization*",
"*io*",
"*route*",
"*timing*"]`?